### DISCUSSION TITLE
宽屏播放, 功能按钮错位

### DISCUSSION THREAD

**Discussion by XMB-7** [the1812/Bilibili-Evolved#3834] [提问 / Q&A]
脚本版本v2.5.2-17-g89d37a1f2. 不过倒是不影响使用. (在讨论区和issue区搜了一下, 好像有人反馈加强版宽屏样式问题, 但不确定说的是不是一件事.)
![image](https://user-images.githubusercontent.com/65625632/203217975-86cdea5e-a4ca-4cb0-b056-64fa5e4f33e2.png)

**Comment by Geeyun-JY3** [comment]
1. 新版播放页未适配：	#2615
2. 你截图里的这个「下载」和「百度云分享」是其他脚本或插件的吧

**Reply by XMB-7** [reply]
确实, 脚本开多了都没注意, 都关了之后ui倒是正常了, 所以这个是要其它脚本那边解决是吧
